dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -35,10 +35,8 @@ def has_required_messages_param?
     params[:messages].present?
   end
 
-  # levelId will be present if we need to get the level's validation files to send along with the
-  # request to openai
-  def has_level_id_param?
```

Comment:
We're sending the `level_id` param for all tutor types now since we want to store that information in the `AITutorInteractions` table, so I had to be more explicit about tracking which levels are validated.